.github/workflows/ci-core.yml

```diff
@@ -47,6 +47,21 @@ jobs:
         run: mkdir -p core/web/assets && touch core/web/assets/index.html
       - name: Download Go vendor packages
         run: go mod download
+      - name: Yarn cache
+        uses: actions/cache@v2
+        env:
+          cache-name: yarn-cache
+        with:
+          path: |
+            ~/.npm
+            ~/.cache
+            **/node_modules
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - run: yarn install --frozen-lockfile
```

Comment:
Need this as core/internal/gethwrappers/go_generate_test.go needs the node modules to build all contracts successfully. Copied this config from solidity build config: https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/solidity.yml#L65-L79